core/ui/src/main/kotlin/nl/q42/template/ui/presentation/SnackbarPresenter.kt

```diff
@@ -0,0 +1,36 @@
+package nl.q42.template.ui.presentation
+
+import de.palm.composestateevents.StateEventWithContent
+import de.palm.composestateevents.consumed
+import de.palm.composestateevents.triggered
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.update
+
+/**
+ * Presents SnackBars using One-Time-UI-Events.
+ *
+ * More info  on One-Time-UI-Events: https://github.com/leonard-palm/compose-state-events
+ */
+interface SnackBarPresenter {
+
+    val snackBarEvents: StateFlow<StateEventWithContent<SnackBarMessageViewState>>
+
+    fun onSnackBarMessageShown()
+    fun showSnackBarMessage(state: SnackBarMessageViewState)
+}
+
+data class SnackBarMessageViewState(val title: ViewStateString)
```

Comment:
FYI: Since this is just a primitive value wrapper class, you can use a value class to reduce boxing/unboxing operations